Crane/app/src/main/java/androidx/compose/samples/crane/calendar/Calendar.kt

```diff
@@ -78,7 +83,7 @@ fun Calendar(
 
 @Composable
 private fun MonthHeader(modifier: Modifier = Modifier, month: String, year: String) {
-    Row(modifier = modifier) {
+    Row(modifier = modifier.clearAndSetSemantics { }) {
```

Comment:
It might be nice to add comments to this saying where the semantics will be set or why you want this to be cleared out